prime-dotnet-webapi/ViewModels/SiteRegistration/OrganizationListViewModel.cs

```diff
@@ -41,6 +41,12 @@ public IEnumerable<string> MatchedOn(string textSearch)
                 matchedOn.Add(nameof(Name));
             }
 
+            if (SigningAuthority != null && (SigningAuthority.FirstName + " " + SigningAuthority.LastName).Contains(textSearch))
+            {
+                matchedOn.Add(SigningAuthority.FirstName);
+                matchedOn.Add(SigningAuthority.LastName);
```

Comment:
This is just the name of the field, it doesn't make sense for the object to come back and say
`Matched on: James`
This can just be `matchedOn.Add("SigningAuthorityName");` or something like that

Reply:
Changed, did it as the other ones with `matchedOn.Add(nameof(SigningAuthority))`

Reply:
Changed